companion/lib/Preview/Graphics.ts

```
@@ -167,20 +166,21 @@ export class PreviewGraphics {
 						const parser = self.#controlsController.createVariablesAndExpressionParser(controlId, null)
 
 						// Do a resolve of the reference for the starting image
-						const result = ParseInternalControlReference(self.#logger, parser, location, options, true)
+						const locationValue = parser.parseEntityOption(options.location, 'variables') // nocommit - this looks wrong
```

ESLint

[error] Unexpected 'nocommit' comment: 'nocommit - this looks wrong'.

(no-warning-comments)